.evergreen/install_c_driver.sh

```diff
@@ -112,6 +72,30 @@ case "$OS" in
         ;;
 esac
 
+# build libbson
+mkdir cmake_build
+cd cmake_build
+"$CMAKE" -G "$GENERATOR" -DENABLE_MONGOC=OFF $CMAKE_ARGS ..
+"$CMAKE" --build . -- $CMAKE_BUILD_OPTS
```

Comment:
The previous Windows cmake commands also had `--config "Debug"`. I believe Debug may be the default. If we're not sure whether it's safe to omit, I'd lean towards being explicit. In the past, I've seen user reports about Windows bugs linking to both the debug/release forms of the C runtime, which usually appear as non-obvious runtime crashes (here's an example: https://jira.mongodb.org/browse/CXX-1961).

Testing now on my Mac, passing `--config` seems to be ignored. It might be safe to always specify `--config Debug` here, and for the `--build` commands below.

Reply:
Done.